scripts/config/github.ts

```diff
@@ -0,0 +1,102 @@
+import { confirm, input } from '@inquirer/prompts';
+import { UninitializedConfigError } from '../errors';
+import { EnvironmentVariable, McpConfigGroup } from '../types';
+import { openBrowser } from '../utils';
+import { ConfigWithMcpServer } from './config';
+import { log } from '../utils/log';
+import { validateTokenHasCorrectPrefix } from '../utils/string';
+
+export class GithubConfig extends ConfigWithMcpServer {
+  private static privateScopes = ['repo', 'read:org'];
+  private static publicScopes = ['repo:status', 'public_repo'];
+
+  private organization: string | undefined;
+  private token: string | undefined;
+
+  constructor() {
+    super({
+      mcpName: 'github',
+      url: 'http://tiger-gh-mcp-server/mcp',
+      name: 'GitHub',
+      description:
+        'This will configure the Tiger GitHub MCP server (https://github.com/timescale/tiger-gh-mcp-server)',
+    });
+  }
+
+  async collect(): Promise<void> {
+    const hasPrivateAccess = await confirm({
+      message: 'Do you want to include access to private repositories?',
+      default: false,
+    });
+
+    const scopes = hasPrivateAccess
+      ? GithubConfig.privateScopes.join(',')
+      : GithubConfig.publicScopes.join(',');
+
+    const url = `https://github.com/settings/tokens/new?description=Tiger%20Agent&scopes=${scopes}`;
+
+    const shouldOpen = await confirm({
+      message: 'Open GitHub to create personal access token?',
+      default: true,
+    });
+
+    if (shouldOpen) {
+      await openBrowser(url);
+    }
+
+    console.log(
+      `Create a GitHub personal access token with '${scopes}' scopes\n`,
+    );
+
+    this.organization = await input({ message: 'GITHUB_ORG:' });
+    this.token = await input({
+      message: 'GITHUB_TOKEN:',
+      validate: (val) => validateTokenHasCorrectPrefix(val, 'ghp_'),
+    });
```

Comment:
I'd maybe ask the GITHUB_ORG prompt first, before opening the browser to generate the token. It was confusing that it wasn't immediately ready to paste after creating it

<img width="546" height="90" alt="Image" src="https://github.com/user-attachments/assets/2ae039e7-e1a9-4417-b4ba-5175e2582c3f" />